### PR TITLE
[sqllite] Throw ArgumentException during empty text command initialization for reader

### PR DESCRIPTION
**Pull request by MaximLipnin**
Fixes https://github.com/mono/mono/issues/13547

### CODE REVIEW

**Comment by MaximLipnin** [comment]
@monojenkins buidl failed

**Comment by MaximLipnin** [comment]
@monojenkins build failed

File: mcs/class/Mono.Data.Sqlite/Test/SqliteDataReaderTest.cs [hunk 1]
```diff
@@ -41,45 +41,65 @@
...
+
+		[Test]
+		public void EmptyStatementThrowsAgrumentException ()
+		{
+			_conn.ConnectionString = _connectionString;
+			SqliteDataReader reader = null;
+			using (_conn) {
+				_conn.Open ();
+				SqliteCommand cmd = (SqliteCommand) _conn.CreateCommand ();
+				cmd.CommandText = string.Empty;
```
Comment (EgorBo) [inline mcs/class/Mono.Data.Sqlite/Test/SqliteDataReaderTest.cs]: What's going to happen if CommandText is null or whitespace(s) ?

**Comment by MaximLipnin** [comment]
@monojenkins build failed

**Pull request merged**